openstack/02a-MAClow/IEEE802154.c

```diff
@@ -62,7 +62,7 @@ void ieee802154_prependHeader(OpenQueueEntry_t* msg,
    // add termination IE accordingly 
    if (payloadIEPresent == TRUE) {
        ielistpresent = IEEE154_IELIST_YES; 
-       frameVersion  = IEEE154_FRAMEVERSION;
+//       frameVersion  = IEEE154_FRAMEVERSION;

```

Comment:
let's not replace lines by commented lines, but just remove them (applies throughput this PR)